posts/2022-10-21-hydra.md

```diff
@@ -0,0 +1,24 @@
+# High level summary
+
+This week, the hydra team reviewed and addressed several open comments on the
+new HeadV1 specification, completing a list the of identified gaps between
+specification and implementation while doing so. In the wake of the recent
+demonstration of SundaeSwap running their DEX in a Hydra Head, they met with
+them to capture feature ideas & incorporate their feedback on the roadmap, as
+well as potential research avenues.
+
+## What did the team achieve this week
+
+- Had the monthly review meeting, a bit earlier than expected: <https://input-output.atlassian.net/wiki/spaces/HYDRA/pages/3650813953/Monthly+report+-+October+2022>
```

Comment:
@ch1bo - I just realized an implicit assumption that I was making, namely that these updates would not include links to documents which are not accessible by the public. I'm curious what the others think. Is there anyway we could move teh report here, and link to it from confluence?

Reply:
I think we should avoid linking to non-public information indeed.